frontend/components/ClickableUrls/ClickableUrls.tests.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import ClickableUrls from "./ClickableUrls";
+
+const TEXT_WITH_URLS =
+  "Contact your IT administrator to ensure your Mac is receiving a profile that disables advertisement tracking. https://privacyinternational.org/guide-step/4335/macos-opt-out-targeted-ads or https://support.apple.com/en-us/HT202074";
+const URL_1 =
+  "https://privacyinternational.org/guide-step/4335/macos-opt-out-targeted-ads";
+const URL_2 = "https://support.apple.com/en-us/HT202074";
+
+describe("ClickableUrls - component", () => {
+  it("renders text and icon", () => {
```

Comment:
I think we could use a few more test cases here to make sure the regex functions as expected. Maybe some cases like link is at the beginning of the string, only one link, link is only via http, link is abbreviated (e.g. `example.com`).